src/export/ExportFFmpegDialogs.cpp

```diff
@@ -280,9 +280,9 @@ void ExportFFmpegAACOptions::PopulateOrExchange(ShuttleGui & S)
          S.StartMultiColumn(2, wxCENTER);
          {
             S.SetStretchyCol(1);
-            S.Prop(1).TieSlider(
-               XXO("Quality (kbps):"),
-               {wxT("/FileFormats/AACQuality"), 160},320, 98);
+            S.TieSpinCtrl(
+               XXO("Quality (kbps)"),
```

Comment:
Is changing this line is necessary?

Reply:
Nope.  Forgot to undo change.  Will fix it up now.